### PR TITLE
Integrate Mapael js library. See #59.

### PR DESCRIPTION
**Pull request by ChrisHng**

### CODE REVIEW

File: docroot/modules/custom/dct_map/dct_map.module [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Contains hook implementations.
+ */
+
+/**
+ * Implements hook_theme().
```
Comment (lauracernau) [inline docroot/modules/custom/dct_map/dct_map.module]: You can give a more explicit comment.

File: docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\dct_map\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\dct_airport_connections\DataProviderInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\dct_airport_connections;
+
```
Comment (lauracernau) [inline docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php]: The class comment is missing.

File: docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  /**
+   * Returns the array associated to the airport connections map.
+   *
+   * @return array
+   */
+  public function getMap() {
+
+
```
Comment (lauracernau) [inline docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php]: Don't need this extra newline here.

File: docroot/modules/custom/dct_map/dct_map.info.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+name: DCT Map
+type: module
+description: Provides DCT flight connections map
+package: DCT
+core: 8.x
+dependencies:
+  - dct_airport_connections
```
Comment (corneaalina) [inline docroot/modules/custom/dct_map/dct_map.info.yml]: No newline at the end of file.

File: docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php (hunk 2 above)
Comment (corneaalina) [reply docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php]: Missing class comment.

File: docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+namespace Drupal\dct_map\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\dct_airport_connections\DataProviderInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\dct_airport_connections;
+
+class ConnectionsMapController extends ControllerBase {
+
+
```
Comment (corneaalina) [inline docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php]: No need for 2 new lines.

File: docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php (hunk 3 above)
Comment (corneaalina) [inline docroot/modules/custom/dct_map/src/Controller/ConnectionsMapController.php]: No need for 2 new lines.

File: docroot/modules/custom/dct_map/templates/dct-map-connections-map.html.twig [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<div class="mapcontainer">
+    <div class="map">
+        <span>Alternative content for the map</span>
+    </div>
+</div>
```
Comment (corneaalina) [inline docroot/modules/custom/dct_map/templates/dct-map-connections-map.html.twig]: New line missing at the end of file.

**Pull request closed without merging**